### DISCUSSION TITLE
What is a Config.json?

### DISCUSSION THREAD

**Discussion by NightDancers** [drizzle-mizzle/CharacterAI-Discord-Bot#51] [Q&A]
I know how to do everything but find the Config.json file, where is that supposed to be? is it supposed to be somewhere in the discord developer portal? I've already created my discord bot i just need to know where to copy and paste the token

**Comment by drizzle-mizzle** [comment]
In the folder with the bot itself...

**Reply by NightDancers** [reply]
That's the thing, there is no folder i made this on the website not the app i really have no idea what im doing ngl, ive looked it up on youtube but i really cant find any tutorial

**Reply by drizzle-mizzle** [reply]
The very first step in setup guide tells you to download a zip file with the bot application.

**Reply by NightDancers** [reply]
Oh yeah I'm blind or something i never saw that, thank you

**Reply by NightDancers** [reply]
But what do you recommend i use, python or visual studio

**Reply by drizzle-mizzle** [reply]
Recommend for what?..
Python is a language. Visual Studio is an IDE. I don't think you would need any of it.

**Comment by NightDancers** [comment]
![Screenshot 2023-04-03 195302](https://user-images.githubusercontent.com/129767188/229658558-0a31d2c0-13e0-43a1-b79e-6acb43b1bf57.png)
I don't know what im supposed to do to fix this, i went to the marketplace and installed it still doesn't work

**Reply by drizzle-mizzle** [reply]
Just open it with a default notebook / notepad editor (idk how it's called in English versions of Windows)

**Reply by NightDancers** [reply]
So like this?
![notepad](https://user-images.githubusercontent.com/129767188/229791458-2ff18884-bdcb-45af-b60c-d968d27a569f.png)

**Reply by drizzle-mizzle** [reply]
Like this, yeah... Place the tokens and run the app, that's all

**Reply by NightDancers** [reply]
Ok i ran the app but it didn't work or im supposed to open it from somewhere on the last step but i don't know where it is
![windows](https://user-images.githubusercontent.com/129767188/229797044-7b19df5a-e41f-4eb5-bd58-3bee57798d05.png)
![windows 2](https://user-images.githubusercontent.com/129767188/229797063-a69a52a8-8638-4c97-8f6e-b1fb0a7e2c57.png)

**Reply by drizzle-mizzle** [reply]
You need to download this.
![image](https://user-images.githubusercontent.com/55811932/229813409-299a0984-3ef4-4e4e-ad05-c9f3753c06eb.png)

**Reply by NightDancers** [reply]
This is my first time using Github so can you tell me where im supposed to go to find the assets category
![github](https://user-images.githubusercontent.com/129767188/229826764-74c53879-e443-41a1-a0f5-947d650561b1.png)

**Reply by drizzle-mizzle** [reply]
On the link. From the first step. In the setup guide. 
https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/releases/tag/bugfix_1.83_b

**Reply by NightDancers** [reply]
Ok I pasted my discord token and user token into the notepad for config, now when i open it it just repeats this over and over again and nothing happens
![ai](https://user-images.githubusercontent.com/129767188/229850988-195872ed-dd1f-490f-ab62-f2f20ec11ea3.png)
![weird](https://user-images.githubusercontent.com/129767188/229851004-c99139ac-ab39-47d5-8dcd-fc1fc297c4a9.png)
